time time_stamp      # Message 발행 시간 (ros::Time)
int32 target_object  # 대상 객체 H/W Index Enum
int32 action         # 명령 동작 Action Index Enum (홀수만 와야 한다)